### PR TITLE
Add tests for programmatically downloading and uploading packages

### PR DESCRIPTION
**Pull request by yahengsu**
This PR adds tests for the `download` and `upload` methods of the `XRPackage` class. 

The download test downloads a package with a specified `TEST_PACKAGE_HASH` and checks to see if the returned package has the same hash. Similarly, the upload test uploads a test `XRPackage` file and checks to see if the response hash is the same as the package hash.

Part of #64.

### CODE REVIEW

File: tests/download-upload-package.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const withPageAndStaticServer = require('./utils/_withPageAndStaticServer');
+
+test('download package programmatically', withPageAndStaticServer, async (t, page) => {
+  const TEST_PACKAGE_HASH = 'QmesnBdGf4yqtC3DH5d1W3GW4qYCtJr6TVfRGBAwQmyCq3'; //avatar.wbn
+  const response = await page.evaluate(downloadFunction, TEST_PACKAGE_HASH);
+  t.true(response);
+});
+
+test('upload package programmatically', withPageAndStaticServer, async (t, page) => {
+  const response = await page.evaluate(uploadFunction, `${t.context.staticUrl}/assets/avatar.wbn`);
```
Comment (avaer) [inline tests/download-upload-package.js:12]: As a minor optimization, can we use a small package here? Each byte costs bandwidth and time for test runs.

Comment (yahengsu) [reply tests/download-upload-package.js:12]: sure thing!

**Comment by yahengsu** [comment]
For sure, I'll take another look !

**Comment by yahengsu** [comment]
Updated test package to be much smaller.

**Pull request merged**